jaseci_kit/jaseci_kit/modules/tfm_ner/tfm_ner.py

```diff
@@ -36,45 +34,21 @@ def config_setup():
         model_config = json.load(jsonfile)
 
     curr_model_path = model_config["model_name"]
-    load_custom_model(curr_model_path)
+    train_data = []
+    if os.path.exists("train/train.txt"):
```

Comment:
Change from txt to json.
Return error if `extract_entity` is called without any model loaded.
train_model should still work out of the box.